Lib/_colorize.py

```diff
@@ -7,15 +7,19 @@
 
 class ANSIColors:
     BACKGROUND_YELLOW = "\x1b[43m"
+    BOLD_BLUE = "\x1b[1;34m"
     BOLD_GREEN = "\x1b[1;32m"
     BOLD_MAGENTA = "\x1b[1;35m"
     BOLD_RED = "\x1b[1;31m"
+    BOLD_TEAL = "\x1b[1;36m"
```

Comment:
ANSI colour with code 36 is more commonly called cyan: https://en.wikipedia.org/wiki/ANSI_escape_code#Colors


```suggestion
    BOLD_BLUE = "\x1b[1;34m"
    BOLD_CYAN = "\x1b[1;36m"
    BOLD_GREEN = "\x1b[1;32m"
    BOLD_MAGENTA = "\x1b[1;35m"
    BOLD_RED = "\x1b[1;31m"
```